static/script.js

```diff
@@ -110,10 +113,21 @@ function pageSetup() {
             }
         })
 
+        
+
         showUserInfo("");
     });
 }
 
+function set_submit_button_loading(is_loading){
+    if (is_loading){
+        document.getElementById('submit-button').disabled = true;
+    }
+    else{
+        document.getElementById('submit-button').disabled = false;
+    }
+}
+
```

Comment:
IMO this function is overkill, just set the disabled state directly where you are calling this function.